lix/packages/sdk/src/commit.test.ts

```diff
@@ -0,0 +1,174 @@
+import { expect, test } from "vitest"
+import { openLixInMemory } from "./open/openLixInMemory.js"
+import { newLixFile } from "./newLix.js"
+import type { LixPlugin } from "./plugin.js"
+
+test("should be able to add and commit changes", async () => {
+	const mockPlugin: LixPlugin = {
+		key: "mock-plugin",
+		glob: "*",
+		diff: {
+			file: async ({ old, neu }) => {
+				return [
+					!old
+						? {
+								type: "text",
+								operation: "insert",
+								old: undefined,
+								neu: {
+									id: "test",
+									text: "inserted text",
+								},
+							}
+						: {
+								type: "text",
+								operation: "update",
+								old: {
+									id: "test",
+									text: "inserted text",
+								},
+								neu: {
+									id: "test",
+									text: "updated text",
+								},
+							},
+				]
+			},
+		},
+	}
+	const lix = await openLixInMemory({ blob: await newLixFile(), providePlugins: [mockPlugin] })
+
+	const firstRef = await lix.db
+		.selectFrom("ref")
+		.selectAll()
+		.where("name", "=", "current")
+		.executeTakeFirstOrThrow()
+	expect(firstRef.commit_id).toBe("0000-000000-0000")
```

Comment:
I assume that's a special "first commit" id that adheres to uuid format?

Reply:
good point, i make this a valid uuid just to be sure, even though we dont enfornce id formats anywhere at the moment